contrib/wlroots0.16/template.py

```diff
@@ -27,18 +27,19 @@
     "xcb-util-renderutil-devel",
     "xcb-util-wm-devel",
 ]
+replaces = ["wlroots"]
```

Comment:
this is nowhere near enough (e.g. people who currently have wlroots-devel installed explicitly in world will have leftover files)

i'd rather have one of the versions (probably the newer one) provide unversioned names virtually for both the main package and the devel package, which will take care of a proper transition

Reply:
sure, added. the replaces= also wasn't necessary since the files don't conflict with the old ones